### PR TITLE
Create seo-quiz.md

### PR DESCRIPTION
**Pull request by varunsharma6956**

### CODE REVIEW

File: seo-learning/seo-quiz.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+-------------------------------
+
+1. What term is typically used to explain the shuffling of positions in search engine results in between significant updates?
+A. Flux.
+B. Waves.
+C. Swaying.
+D. Shuffling.
+
+
+Answer:- (A)
```
Comment (auchynnikau) [inline seo-learning/seo-quiz.md:11]: Would be great to keep following common style guide for answers using checkbox markdown format. You can look trow how it was implemented in other `quiz.md` files.

File: seo-learning/seo-quiz.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
+Search Engine Optimization(SEO)
```
Comment (auchynnikau) [inline seo-learning/seo-quiz.md:1]: Can you update main table in `README.md` file by including SEO quiz and providing link on it?

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Thank you @varunsharma6956 great job, could you double check if your question already exist in this file: https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/main/seo/search-engine-optimization-quiz.md

**Comment by varunsharma6956** [comment]
Ok sure

**Comment by varunsharma6956** [comment]
Sir my questions and answers doesn't exist on this  link wnich is

https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/main/seo/search-engine-optimization-quiz.md

Can you please approve my PR sir and merge it.

**Comment by shar2040** [comment]
yes sir i double checked mine questions and answers are different one from the link you gave sir can you merge it and approve

**Comment by Ebazhanov** [comment]
> yes sir i double checked mine questions and answers are different one from the link you gave sir can you merge it and approve

@varunsharma6956 well what I mean, you have to not only compare but use the same file and not create your own. Please follow the formatting in already existing quizzes and add your new question accordingly. Thanks

**Comment by varunsharma6956** [comment]
Ok sir but mine questions and answers I have made of seo not taken from
there . If possible merge it

**Comment by shar2040** [comment]
i also used same seo file

**Comment by varunsharma6956** [comment]
sir I have used same file of seo which you told now and added mine in it, now can you see

**Comment by Ebazhanov** [comment]
> sir I have used same file of seo which you told now and added mine in it, now can you see

pls double check your questions should inside https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/blob/main/seo/search-engine-optimization-quiz.md

**Comment by varunsharma6956** [comment]
I have created sir seo-quiz.md inside seo folder inside that , can you
approve pr and approve it I have checked it and inside seo-quiz.md mine 19
questions there with answers

**Comment by Ebazhanov** [comment]
> I have created sir seo-quiz.md inside seo folder inside that , can you approve pr and approve it I have checked it and inside seo-quiz.md mine 19 questions there with answers

thank you @varunsharma6956 for your effort but it does not work like this, I would prefer to keep everything in one file. And not create several. By the end it will bring complexity. Can you merge it in main file please or I can do it. Please let me know if you need any help. I still believe we can have your new questions here 👍

**Comment by varunsharma6956** [comment]
Sir can you convinced file and merge it.

**Comment by shar2040** [comment]
Sir can you put into folder and merge file and  accept pr

**Comment by Ebazhanov** [comment]
> Sir can you put into folder and merge file and accept pr

will do, please give me some time I will create PR and assign you for review!

**Comment by auchynnikau** [comment]
> Sir can you put into folder and merge file and accept pr

@shar2040 if you want, you can invite me to collaborate and I can help you to do it within you PR

**Comment by auchynnikau** [comment]
> Sir can you put into folder and merge file and accept pr

@shar2040 if you want, you can invite me as a collaborator to your repo and I'll help you to finish this issue within your PR

**Pull request closed without merging**